packages/ndla-i18n/src/i18n.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+const DETECTION_OPTIONS = {
+  order: ['localStorage', 'path', 'htmlTag'],
+  caches: ['localStorage'],
+  lookupLocalStorage: 'i18nextLng',
+};
+
+const lng = ['nn', 'nb', 'en'];
+
+export const isValidLocale = (locale: string) => lng.find(l => l === locale) !== undefined;
```

Comment:
Kan bare bruke includes() her hvis jeg har forstått funksjonen riktig:
```suggestion
export const isValidLocale = (locale: string) => lng.includes(locale);
```
Kanskje subjektivt, men jeg foretrekker også litt mer beskrivende variabelnavn som validLanguages eller noe sånt i stedet for lng